src/components/Admin.js

```diff
@@ -16,13 +16,19 @@ import FAQs from './FAQs'
 import FAQDetails from './FAQDetails'
 import FAQAnswers from './FAQAnswers'
 import FAQAnswerDetails from './FAQAnswerDetails'
+import Users from './Users'
+import UserDetails from './UserDetails'
 
 const Admin = () =>
     <div>
         <h2>Admin</h2>
         <Router>
             <div className="row">
                 <div className="col-3">
+                    <Link to="/admin/users">Users</Link>
+                    <br/>
+                    <Link to="/admin/users/1">User Details</Link>
+                    <br/>
```

Comment:
You have to add the routes down at the bottom too so it knows what component to go to when you go to the links.